components/src/core/Drawer/DrawerHeader/DrawerHeader.tsx

```diff
@@ -72,8 +72,6 @@ export type DrawerHeaderProps = ToolbarProps & {
 const Root = styled(Toolbar, { name: 'drawer-header', slot: 'root' })<
     Pick<DrawerHeaderProps, 'backgroundColor' | 'fontColor'>
 >(({ backgroundColor, fontColor, theme }) => ({
-    paddingRight: 0,
```

Comment:
Was the default padding already 0?  Why was this removed?

Reply:
As paddingRight and paddingLeft are removed as disableGutters is handling the padding.

Reply:
I'm fine with this approach, but it could have been done by adding the following to the Root styled component:
`['&.MuiToolbar-root']: {
        paddingRight: 0,
        paddingLeft: 0,
    },`